### PR TITLE
Dive request

### PR DESCRIPTION
**Pull request by ahohnsen**

### CODE REVIEW

File: src/components/RequestForm.spec.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  it('renders a form with three input fields, a textarea and a button to submit the form', () => {
+    render(
+      <MemoryRouter>
+        <RequestForm buttonName="POST" />
+      </MemoryRouter>
+    );
+
+    const destinationInput = screen.getByLabelText(/destination/i);
+    const descriptionTextarea = screen.getByLabelText(/description/i);
+    const submitButton = screen.getByRole('button', { name: 'POST' });
```
Comment (mwarnck) [inline src/components/RequestForm.spec.js:15]: You could test for the date inputs too ;)

**Review by mwarnck — APPROVED** [review]
Looks beautiful, very impressive! :)

**Review by mwarnck — APPROVED** [review]
Looks like just small changes that fix this huge bug :) Great work!

File: src/components/RequestForm.spec.js (hunk 1 above)
Comment (ahohnsen) [reply src/components/RequestForm.spec.js]: Thx, have added that :)

File: src/components/IconButton.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 
 const IconButton = styled.button`
-  position: absolute;
+  /* position: absolute; */
```
[the remark this answers is not in the document]
Comment (ahohnsen) [reply src/components/IconButton.js]: Haha, yes, thanks

**Pull request merged**